.github/workflows/node.yml

```diff
@@ -13,7 +13,7 @@ jobs:
         with:
           versionsAsRoot: true
           type: 'majors'
-          preset: '>=10'
+          preset: '>=17'
```

Comment:
this file still needs to be largely reverted

Reply:
I has fixed